### ISSUE TITLE
not allowed to perform gas push

### ISSUE THREAD

**Issue by Identity-labs** [MaartenDesnouck/google-apps-script#117]
Hi,

I can't push any projects, but I can clone and pull.

```
gas push
Pushing 'myposeo_dev' to remote ... [✘]
You are not allowed to perform this action.
```

Environnement :

Mac OS X El Capitan 10.11.6

node and npm installed with macport

```
sudo port install nodejs8
sudo port install npm4
```

```
node -v
v8.11.3
```

```
npm -v
4.6.1
```

Thx

**Comment by MaartenDesnouck** [comment]
Yes, seeing your error in the logs. Seems like you don't have edit right to this file, is that possible?
If you open the project in the online editor (```gas open``) are you able to make changes then?

If you are able, then I would try to do ```gas auth -f``` to force reauthentication and try again...

**Comment by Identity-labs** [comment]
It's not possible to have a full stacktrace ?

**Comment by MaartenDesnouck** [comment]
Ah wait I've found it: 
"User has not enabled the Apps Script API. Enable it by visiting https://script.google.com/home/usersettings then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry."

**Comment by MaartenDesnouck** [comment]
Ah no you haven't... That's new since 4.0.0, will add it to the readme.

**Comment by Identity-labs** [comment]
It's good now.
I think we can add this in the README for the futur :)
Thx for your help and your quick reply too 👍

**Comment by MaartenDesnouck** [comment]
Fixed in 4.1.1
